<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loan</class>
 <widget class="QWidget" name="loan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Loan</string>
  </property>
  <widget class="QLabel" name="amount_text">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>182</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">Amount per month</string>
   </property>
  </widget>
  <widget class="QLabel" name="amount_output">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>184</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="account_output">
   <property name="geometry">
    <rect>
     <x>278</x>
     <y>97</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="cpf_field_input">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>11</y>
     <width>170</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>11</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="name_output">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>68</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cpf_text_output">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>93</y>
     <width>45</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">CPF</string>
   </property>
  </widget>
  <widget class="QLabel" name="cpf_output">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>96</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="error_message_cpf">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>39</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="search_button">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>11</y>
     <width>90</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_text">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>66</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="account_text">
   <property name="geometry">
    <rect>
     <x>219</x>
     <y>96</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">Account</string>
   </property>
  </widget>
  <widget class="QLabel" name="cpf_text_input">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>15</y>
     <width>44</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">CPF</string>
   </property>
  </widget>
  <widget class="QLabel" name="resquested_amount_text">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>124</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">Requested amount</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="resquested_amount_input">
   <property name="geometry">
    <rect>
     <x>138</x>
     <y>127</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>MAX 9999999.99</string>
   </property>
  </widget>
  <widget class="QLabel" name="fees_text">
   <property name="geometry">
    <rect>
     <x>223</x>
     <y>151</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">Feeds  (%) a.m</string>
   </property>
  </widget>
  <widget class="QLabel" name="fees_output">
   <property name="geometry">
    <rect>
     <x>308</x>
     <y>154</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="payment_installments_text">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>152</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">Payment installments</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="payment_installments_input">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>154</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_input">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>217</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="password_text">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>216</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Customer's password</string>
   </property>
  </widget>
  <widget class="QLabel" name="payday_text">
   <property name="geometry">
    <rect>
     <x>271</x>
     <y>182</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">Payday</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="payday_intput">
   <property name="geometry">
    <rect>
     <x>318</x>
     <y>184</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="loan_button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>284</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Loan</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_message">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>250</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>242</x>
     <y>284</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="check_button">
   <property name="geometry">
    <rect>
     <x>307</x>
     <y>212</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: orange;
 border-style: solid;
color: black;
border-radius: 30px</string>
   </property>
   <property name="text">
    <string>Check</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cpf_field_input</tabstop>
  <tabstop>search_button</tabstop>
  <tabstop>resquested_amount_input</tabstop>
  <tabstop>payment_installments_input</tabstop>
  <tabstop>payday_intput</tabstop>
  <tabstop>password_input</tabstop>
  <tabstop>check_button</tabstop>
  <tabstop>loan_button</tabstop>
  <tabstop>cancel_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
